mod/deckorate_core/set/type/metric/export.rb

```diff
@@ -108,3 +108,65 @@ def semicolon_separated_values column
     card.send("#{column}_card").item_names.join ";"
   end
 end
+
+format :jsonld do
+  
+  def molecule
+    metric_jsonld(atom)
+  end
+
+  private
+
+    def metric_jsonld(a)
+    {
+      "@context" => "#{request.base_url}/context/#{card.type}.jsonld",
+      "@id"      => path(mark: card.name, format: nil),
+      "@type"    => card.type,
+
+      "title"    => a[:title] || a["title"],
+      "designer" => "#{request.base_url}/#{a[:designer] || a["designer"]}",
+      "question"  => a[:headquarters] || a["headquarters"],  
+
+      "metric_type"                 => a[:metric_type] || a["metric_type"],
+      "research_policy"             => a[:research_policy] || a["research_policy"],
+      "about"                       => ActionView::Base.full_sanitizer.sanitize(a[:about] || a["about"]),
+      "methodology"                 => ActionView::Base.full_sanitizer.sanitize(a[:methodology] || a["methodology"]),
```

Comment:
a sanitize method in this class that calls the ActionView sanitizer would be cleaner